src/features/loans/components/MyAPYCell.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useMemo, useCallback } from 'react';
+import BN from 'bn.js';
+
+import { useMyInterestShare, useMyStakeCost } from 'features/stake';
+import { PercentAmount } from 'model/entities';
+import { Loading, FormattedAmount, DoubleLineCell } from 'components';
+import { calcLoanAPY, calcLoanProfit } from 'domainLogic';
+
+import { PartialDebt } from './types';
+
+type Props = {
+  loanAPR: PercentAmount;
+  account: string;
+  debt: PartialDebt;
+};
+
+export function MyAPYCell({ loanAPR, account, debt }: Props) {
```

Comment:
Have you used the formatter? As I remember it should warn you that you should destructure `props` inside the function's body